### PR TITLE
doc: fix port-forward command in run-inference.md

### PR DESCRIPTION
**Pull request by ckadner**
#### Motivation

The code snippet to `port-forward` the `modelmesh-serving` service did not work:

```Bash
$ kubectl port-forward modelmesh-serving 8033:8033

Error from server (NotFound): pods "modelmesh-serving" not found
```

#### Modifications

Update the code snippet to specify the resource type `service`:

```Bash
$ kubectl port-forward service/modelmesh-serving 8033:8033
```

#### Result

```
Forwarding from 127.0.0.1:8033 -> 8033
Forwarding from [::1]:8033 -> 8033
Handling connection for 8033

```

/cc @njhill 
/cc @chinhuang007

### CODE REVIEW

**Review by chinhuang007 — APPROVED** [review]
Looks good, thanks, @ckadner !

**Comment by njhill** [comment]
Not sure why the bot isn't merging these... I'll do it manually.

**Comment by ckadner** [comment]
> Not sure why the bot isn't merging these... I'll do it manually.

I think the bot was waiting for both `/lgtm` and `/approve` labels. Your PRs automatically get the `/approve` labels as you are a project committer, so anyone reviewing your PRs only needs to add a `/lgtm`.

**Pull request merged**
